packages/by-name/runtime-class-files/package.nix

```diff
@@ -10,13 +10,13 @@
 let
   rootfs = microsoft.kata-image;
   igvm = if debugRuntime then microsoft.kata-igvm.debug else microsoft.kata-igvm;
-  cloud-hypervisor-bin = "${microsoft.cloud-hypervisor}/bin/cloud-hypervisor";
+  cloud-hypervisor = "${microsoft.cloud-hypervisor}/bin/cloud-hypervisor";
```

Comment:
`-bin` is commonly reserved for things that are packaged from binary instead of source. That was also the original intention when naming this (not the subpath).